.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -26,7 +26,7 @@ Steps to reproduce the behavior:
 
 <!-- If applicable, add screenshots to help explain your problem. -->
 
-**Please paste the results of `webpack-cli info` here, and mention other relevant information**
+**Please paste the results of `webpack-cli info` and `webpack-cli version` here, and mention other relevant information**
```

Comment:
Is it `webpack-cli info` is not output versions?

Reply:
Actually, it does. Don't know why it didn't when I tested last time (maybe I wasn't in the correct repository while testing earlier)

Reply:
yep, I think `webpack-cli info` is enough, ideally we should ask this output in all issue templates, better to use `npx --no-install webpack-cli info`

Reply:
I will update.